server/requestResponseHandler.go

```diff
@@ -0,0 +1,12 @@
+package server
```

Comment:
regular file names are lower case, short, and without any sort of underscore or space. Generally, file names follow the same convention as package names. 
https://golang.org/doc/effective_go.html#package-names